src/Symfony/Component/HttpKernel/Tests/EventListener/ServiceResetListenerTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Symfony\Component\HttpKernel\Tests\EventListener;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\Argument\IteratorArgument;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\HttpKernel\EventListener\ServiceResetListener;
+use Symfony\Component\HttpKernel\Tests\Fixtures\ClearableService;
+use Symfony\Component\HttpKernel\Tests\Fixtures\ResettableService;
+
+class ServiceResetListenerTest extends TestCase
+{
+    /**
+     * @before
```

Comment:
We never used this annotation in Symfony core (AFAIK). Shouldn't we use `setUp` as usual?

Reply:
All right. Changed.